.evergreen/config.yml

```diff
@@ -434,6 +441,17 @@ tasks:
         permissions: public-read
         local_file: 'libmongocrypt-java.tar.gz'
         content_type: '${content_type|application/x-gzip}'
+    - command: s3.put
```

Comment:
What does this command do if `libmongocrypt-java-${tag_upload_location}.tar.gz` does not exist (since it's created conditionally, above)?  I'm guessing some sort of error, but does it cause any sort of abort or global task failure?

Reply:
This is copied from the libmongocrypt-all uploading and it should just log a message like it does with the libmongocrypt-all upload:

```
file '/data/mci/c2027b19a6d6d313601aa5939dfa35e0/libmongocrypt-all-.tar.gz' not found but skip missing true
```